charmcraft/providers.py

```diff
@@ -36,10 +37,11 @@
 from charmcraft.snap import get_snap_configuration
 
 
-BASE_CHANNEL_TO_PROVIDER_BASE = {
-    "18.04": bases.BuilddBaseAlias.BIONIC,
-    "20.04": bases.BuilddBaseAlias.FOCAL,
-    "22.04": bases.BuilddBaseAlias.JAMMY,
+BASE_NAME_VERSION_TO_PROVIDER_BASE = {
```

Comment:
opinion: Consider the tuple a "charmcraft base" and call this `CHARMCRAFT_BASE_TO_PROVIDER_BASE`, which is more clear than using each part of the tuple itself (`NAME` and `VERSION`)

Reply:
Moved to craft-providers.